stdlib/public/core/SmallString.swift

```diff
@@ -346,13 +346,22 @@ extension _SmallString {
   //
   @_effects(readonly) // @opaque
   @usableFromInline // testable
-  internal init(taggedCocoa cocoa: AnyObject) {
+  internal init?(taggedCocoa cocoa: AnyObject) {
```

Comment:
Are we certain that this isn't ever used from an inlinable public API?

Reply:
We are. It *is*, however, used from test code, which I had to fix up.